yaiechnyk-oleh/flask-hello-world-application
# Add  /api/v1/hello-world-21 route to Flask app

## Changes Introduced

- Implemented a new API route `/api/v1/hello-world-21` that returns a "Hello World 21" response with an HTTP status code of 200.

## Purpose

The purpose of this pull request is to integrate a new route into the Flask application which is part of the initial setup for our application. This route will serve as a template for future API endpoints.

## How to Test

1. Check out to the `feature/hello-world-21-route` branch.
2. Activate the virtual environment and install dependencies with `pipenv install`.
3. Run the Flask application using `flask run`.
4. Navigate to `http://127.0.0.1:5000/api/v1/hello-world-21` in a web browser or use a tool like `curl` to make a GET request.
5. Confirm that the response is "Hello World 21" with a status code of 200.

## Additional Information

- Added a `requirements.txt` file generated from the current environment with `pipenv lock`.
- Created a `README.md` with setup and run instructions
